ui/eureka/eureka.js

```diff
@@ -404,6 +405,22 @@ class EurekaTracker {
       if (match) {
         this.currentTracker = match[1];
       }
+      match = log.match(gImportRegex);
+      if (match) {
+        var webKeys = ['Sabo', 'Lord', 'Teles', 'Emperor', 'Callisto', 'Number', 'Jaha', 'Amemet', 'Caym', 'Bomba', 'Serket', 'Julika', 'Rider', 'Poly', 'Strider', 'Hazmat', 'Fafnir', 'Amarok', 'Lamashtu', 'Pazuzu'];
```

Comment:
Can you move these up to zone info dict up top? Maybe just add a trackerKey field to each of them with this string, e.g.
```
lama: {
  name: 'Lama',
  mobName: 'Lamashtu',
  trackerName: 'Lamashtu',
  x: 7.7,
  y: 25.3,
  time: 'Night',
},
```

There's going to be some other Eureka zone soon, so having it in the zone info will make it easier to add the next zone.

Reply:
@quisquous I thought about doing it that way, but I wasn't sure how to get the names from there without doing a for loop and looking at each one individually. 
I can do it that way if you want or another option would be to change your keys to match theirs if you aren't attached to them. Then we can just do a toLower on the string to match them up.